R/metab.bookkeep.R

```diff
@@ -73,7 +73,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
 		warning("datetime not found, inferring sampling frequency from # of observations") # issue a warning (note checks in addNAs)
 		# NOTE: because of the checks in addNA's, it is unlikely a user would receive this warning via metab()
 		# warning will only be seen through direct use of metab.bookkeep when datettime is not supplied
-		freq <- nobs
+		freq <- nobs - 1
```

Comment:
Late to the discussion, but yeah currently metab() is only [passing in one midnight](https://github.com/GLEON/LakeMetabolizer/blob/master/R/metab.R#L170). Could you just grab the next measurement (ids == i + 1 & time == "00:00:00"), rbind that with the previous day, and pass it all to the specific metabolism function? I think that should work with bookkeeping, not familiar with the other methods.